pkg/broker/rbac.go

```diff
@@ -81,7 +81,7 @@ func NewBrokerClusterRole() *rbacv1.Role {
 			rbacv1.PolicyRule{
 				Verbs:     []string{"create", "get", "list", "watch", "patch", "update", "delete"},
 				APIGroups: []string{"submariner.io"},
-				Resources: []string{"clusters", "endpoints"},
+				Resources: []string{"clusters", "endpoints", "gateways"},
```

Comment:
Those rbacs are for the broker cluster, we should not handle gateways on the broker cluster.